### PR TITLE
Feature cassiopeia/remove obsolete margin

### PR DESCRIPTION
**Pull request by hans2103**
Pull Request for Issue https://github.com/joomla/joomla-cms/pull/31195

### Summary of Changes

After merge of https://github.com/joomla/joomla-cms/pull/31195 this PR will fix obsolete spacing between elements.
It removes the margin-bottom from both .breadcrumb and empty alert box. And it removes the empty alert box from the DOM.

### Testing Instructions

- wait until https://github.com/joomla/joomla-cms/pull/31195 is merged
- Joomla 4 test and press login button on right sidebar
- It will take you to index.php/author-login with an alert message
- Close the alert message by pressing the `x`
- Notice the gap between the breadcrumbs and the output of the component. 
- Apply patch, run `npm run build:css` and repeat the steps above

### Expected result

The expected result is a gap of 1rem. Breadcrumb and the output of the component have no obsolete margin between the elements. 

Schermafbeelding 2020-10-22 om 08 56 22

### Actual result

The actual result is a gap bigger than 1rem. This is due to a margin-bottom on the breadcrumbs and a margin-top added to the empty alert box, although it is empty. It is a succeeding element and therefor it gets a margin-top.

Schermafbeelding 2020-10-22 om 08 57 14

### CODE REVIEW

**Review by drmenzelit — APPROVED** [review]

**Comment by richard67** [comment]
@drmenzelit @hans2103 As the description says, this PR has to wait until joomla/joomla-cms#31195 is merged in the CMS repo. But now as this PR here has been approved, it is not protected anymore from being merged. So if someone of us works on a touch screen and falls asleep due to too much work and then accidently hits the merge button with his or her forehead, the PR gets merged.

I am not happy with that.

To be safe, @hans2103 could change the PR back to draft.

**Comment by hans2103** [comment]
converted to draft

**Comment by drmenzelit** [comment]
Sorry for being too fast ;-)

**Comment by hans2103** [comment]
Since  joomla/joomla-cms#31195 has been merged, this PR is no longer a draft.

**Comment by chmst** [comment]
Merged now, thanks!

**Pull request merged**
